pkg/components/standalone_loader.go

```diff
@@ -37,23 +44,93 @@ func (s *StandaloneComponents) LoadComponents() ([]components_v1alpha1.Component
 	list := []components_v1alpha1.Component{}
 
 	for _, file := range files {
-		if !file.IsDir() {
+		if !file.IsDir() && s.IsYaml(file.Name()) {
 			b, err := ioutil.ReadFile(fmt.Sprintf("%s/%s", dir, file.Name()))
-			if err != nil {
-				log.Warnf("error reading file: %s", err)
-				continue
-			}
 
-			var component components_v1alpha1.Component
-			err = yaml.Unmarshal(b, &component)
 			if err != nil {
-				log.Warnf("error parsing file: %s", err)
+				log.Warnf("error reading file %s/%s : %s", dir, file.Name(), err)
 				continue
 			}
 
-			list = append(list, component)
+			components, _ := s.DecodeYaml(fmt.Sprintf("%s/%s", dir, file.Name()), b)
+			list = append(list, components...)
 		}
 	}
 
 	return list, nil
 }
+
+// IsYaml checks whether the file is yaml or not
+func (s *StandaloneComponents) IsYaml(fileName string) bool {
```

Comment:
Can you please lowercase new method names?  because these are used only in this file

Reply:
Pretty new to "Go". Thanks for the comment. Done the changes